genesis/ext/pyrender/mesh.py

```diff
@@ -266,7 +266,7 @@ def _get_trimesh_props(mesh, smooth=False, material=None):
                     else:
                         colors = vc[mesh.faces].reshape((3 * len(mesh.faces), vc.shape[1]))
                 material = MetallicRoughnessMaterial(
-                    alphaMode="OPAQUE" if (colors[..., 3] == 255).all() else "BLEND",
+                    alphaMode="OPAQUE" if colors.shape[-1] < 4 or (colors[..., 3] == 255).all() else "BLEND",
```

Comment:
not sure if this is a correct fix. Basically, we will get an error if the color argument in draw_debug_arrow takes (r,g,b) instead of (r,g,b,a). You can reproduce it by reverting this change and running @duburcqa 
```
pytest -s -v tests/test_rigid_physics.py::test_examples_api
```